spec/02-integration/06-invalidations/03-plugins_map_invalidation_spec.lua

```diff
@@ -0,0 +1,262 @@
+local cjson        = require "cjson"
+local helpers      = require "spec.helpers"
+
+
+local POLL_INTERVAL = 0.3
+
+
+for _, strategy in helpers.each_strategy() do
+  describe("plugins map with db [#" .. strategy .. "]", function()
+
+    local admin_client_1
+    local admin_client_2
+
+    local proxy_client_1
+    local proxy_client_2
+
+    local wait_for_propagation
+
+    local service_fixture
+
+    setup(function()
+      local bp = helpers.get_db_utils(strategy, {
+        "apis",
+        "routes",
+        "services",
+        "plugins",
+        "certificates",
+      })
+
+      -- insert single fixture Service
+      service_fixture = bp.services:insert()
+
+      local db_update_propagation = strategy == "cassandra" and 0.1 or 0
+
+      assert(helpers.start_kong {
+        log_level             = "debug",
+        prefix                = "servroot1",
+        database              = strategy,
+        proxy_listen          = "0.0.0.0:8000, 0.0.0.0:8443 ssl",
```

Comment:
side note: ouch, it is a realization to me that other existing test suites already make use of the default ports used by Kong, which breaks the test suite when a development instance is running... No action required here, just a note...

Reply:
Yes, I have hit this many times :-(.

Reply:
This test was a copypasta of the boilerplate code in https://github.com/Kong/kong/blob/next/spec/02-integration/06-invalidations/02-core_entities_invalidations_spec.lua.

Reply:
Yep, that's how I realized we were already using those ports for existing test suites.